.travis.yml

```diff
@@ -20,16 +35,26 @@ deploy:
   # Push image to Dockerhub on merge to master
   - provider: script
     skip_cleanup: true
-    script: docker push nfvpe/sriov-cni
+    script: >
+      bash -c '
+      if [ "${ARCH}" == "amd64" ]; then
+        docker push nfvpe/sriov-cni
+      fi
+      docker tag nfvpe/sriov-cni nfvpe/sriov-cni:$ARCH;
+      docker push nfvpe/sriov-cni:$ARCH;
+      echo foo'
     on:
       branch: master
   # Push image to Dockerhub on tag
   - provider: script
     skip_cleanup: true
     script: >
       bash -c '
-      docker tag nfvpe/sriov-cni nfvpe/sriov-cni:$TRAVIS_TAG;
-      docker push nfvpe/sriov-cni:$TRAVIS_TAG;
+      if [ "${ARCH}" == "amd64" ]; then
+        docker push nfvpe/sriov-cni:$TRAVIS_TAG
```

Comment:
Does the tag build have TRAVIS_TAG appended to sriov-cni image automatically? or do we need to tag the image to TRAVIS_TAG?

Reply:
It should be add by the script, seems I removed the line docker tag nfvpe/sriov-cni nfvpe/sriov-cni:$TRAVIS_TAG; by mistake